.github/pull_request_template.md

```diff
@@ -1,29 +1,40 @@
-<!--
-## Tips for pull requests
-* use chat, discussions, etc... to refine an idea for big and/or impactful changes
-* open them relatively early in draft mode and get regular feedback
-* add notes for reviewers and yourself as you go, so it's easy to maintain context
-* refine the pull request message over time
--->
+## Summary
+* what changed and how?
+* why are we changing it?
 
-<!--
-## Finalizing a PR:
+## Details
+* info that couldn't fit into the summary
+* relevant details; tricky parts
+* anything else
 
-### title:
-* reads like a short changelog line
+Fixes #(issue)
 
-### body:
-* describe the current behaviour
-* describe why this particular approach
-* if it's breaking, migration steps
-* note any follow-on work
-
-### content:
-* leave code better than you found it, add docs, tests, etc
--->
+---
+<!-- Note: section break (`---`) onwards is not in CI merge commit -->
 
+## Notes for Reviewers
+* additional context for reviewers
+* maybe specific requests or areas of focus
 
 <!--
-* note any issues that this fixes entirely
-* use `fixes #123 .` to auto-close issues
--->
\ No newline at end of file
+Pull Request(PR) Help
+
+Before Merge Ensure:
+* title reads like a short changelog line entry
+* code is includes tests and is documented
+* leaves the source better than before, but split out big reformats
+
+See contributor (guide)[https://nim-works.github.io/nimskull/contributing.html]
+for details, especially if you're new to this project.
+
+Tips that make PRs easier:
+* for big/impactful changes, start with chat/discussions to refine ideas
+* open them relatively early in draft mode and get regular feedback
+* refine the pull request message over time; don't have to nail it in one go
+* handle the single commit message requirement at the end of review
+* note any related issues, for fixed issues use `fixes #123.` to auto-close
+
+How to ease the PR process:
+* collaborate: an ambitious volunteer run project needs some processes
+* kindness: we want to avoid open source maintainer burnout
+* stay focused: try to keep changes small so they can move quickly
```

Comment:
keep the PR template as little as possible. Only put in the essential information and constraints. Tips and how-to instructions should be placed elsewhere (e.g. in the linked contibuting guideline). Otherwise the template will be overwhelming (and ignored).

Reply:
It's a balance, plenty of people could be entirely new to it and it's their first time.

I agree that it needs to shrink, I think a contribution quick start is needed between PR template and the current overwhelming contribution guide.

I'll leave this here for this round and cleanup the contribution guide, setup a quick start, and drop out this part later in the week. Sound good?